trace_context_ext/HTTP_HEADER_FORMAT_RATIONALE.md

```diff
@@ -0,0 +1,27 @@
+#  Trace Context Extended HTTP Header Format Rationale
```

Comment:
hints we may eventually need to extract a definition of the comma separatey format as it would mostly be reused for trace-context-ext and correlation-context so far.